Makefile

```diff
@@ -53,7 +53,9 @@ $(eval $(call build-package,mpc,1.2.1-r2))
 $(eval $(call build-package,isl,0.24-r2))
 $(eval $(call build-package,zlib,1.2.13-r1))
 $(eval $(call build-package,flex,2.6.4-r2))
-$(eval $(call build-package,glibc,2.36-r3))
+$(eval $(call build-package,python-3.11,3.11.1-r0))
```

Comment:
I'm not happy about moving this higher in the Makefile, but it was needed since glibc now depends on system-python to build.